social/src/main/java/org/exoplatform/platform/qa/ui/social/pageobject/MyNotificationsSetting.java

```diff
@@ -291,7 +291,7 @@ public void enableNotification(myNotiType notifToEnable, Object... opParams) {
         evt.check(ELEMENT_EDIT_NEWUSER_WEB_CHECKBOX, 2);
         info("Click on Save button");
         $(ELEMENT_EDIT_NEWUSER_SAVE_BTN).click();
-        $(ELEMENT_EDIT_NEWUSER_SAVE_BTN).waitUntil(Condition.not(Condition.visible),Configuration.timeout);
+        $(ELEMENT_EDIT_NEWUSER_SAVE_BTN).waitUntil(Condition.visible,Configuration.timeout);
```

Comment:
Why did you make this change ?  "$(ELEMENT_EDIT_NEWUSER_SAVE_BTN)" must be not visible after click.

Reply:
done